### PR TITLE
Optional layers

### PR DESCRIPTION
**Pull request by deepio**

### CODE REVIEW

File: run_calvo_trainer_mq.py [hunk 1]
```diff
@@ -63,74 +63,81 @@ def download_resource(path, url, headers):
...
-    with open(SM_RES, 'rb') as f:
-        results["Staff Lines Model"] = base64.encodebytes(f.read()).decode("utf-8")
-    with open(TM_RES, 'rb') as f:
-        results["Text Model"] = base64.encodebytes(f.read()).decode("utf-8")
+
+    # Send response (for Optional layers) "Model %d"
+    for k in body["outputs"]:
+        if k[:5] == "Model":
+            l = k[6:7]
+            model = 'Model %d' % l
```
Comment (JRegimbal) [inline run_calvo_trainer_mq.py:135]: Can you use clearer variable names here?

File: run_calvo_trainer_mq.py [hunk 2]
```diff
@@ -63,74 +63,81 @@ def download_resource(path, url, headers):
...
+            model_file = "model_%d.png" % l
+
+            # Declare input filepath
+            inputs['rgba PNG - Layer %d' % l] = os.path.join(slurm_dir, layer_file)
+
+            # Download File
+            download_resource(layer_file, base_url + body["inputs"]["rgba PNG - Layer %d" % l], headers)
+
+            # Declare output filepath
+            outputs['Model %d' % l] = os.path.join(slurm_dir, model_file)
```
Comment (JRegimbal) [inline run_calvo_trainer_mq.py:116]: Clearer names for `k` and `l`.

**Review by JRegimbal — CHANGES_REQUESTED** [review]
Overall looks fine, just I think some variables should have more descriptive names so it's clearer for whoever looks at this next. Unrelated, but do the type annotations add runtime checking?

File: run_calvo_trainer_mq.py (hunk 2 above)
Comment (deepio) [reply run_calvo_trainer_mq.py]: resolved with: 6b5ec20

**Comment by deepio** [comment]
> Overall looks fine, just I think some variables should have more descriptive names so it's clearer for whoever looks at this next. Unrelated, but do the type annotations add runtime checking?

The Python runtime does not enforce function and variable type annotations. I think of it like a comment which makes it easier for the next person to understand what's going on. You instead check for problems using something like mypy.

**Review by JRegimbal — APPROVED** [review]

**Pull request merged**